dll/win32/browseui/parsecmdline.cpp

```diff
@@ -382,8 +382,43 @@ SHExplorerParseCmdLine(ExplorerCommandLineParseResults * pInfo)
             {
                 // Or just a plain old string.
 
+                WCHAR szPath[MAX_PATH];
+
+                if (!wcsncmp(strField, L"\\", 1))
```

Comment:
No point in calling wcsncmp if length is 1, just compare the char directly.

Reply:
I tried to do `strField[0] == L"\\"` but it results in compile error `comparison with string literal results in unspecified behaviour [-Werror=address]`.

Reply:
Yes, you need `L'\\'`, i.e. single quotes.